subsys/bluetooth/host/hci_core.c

```diff
@@ -7447,9 +7451,12 @@ int bt_le_adv_start_ext(struct bt_le_ext_adv *adv,
 
 			bt_conn_set_state(conn, BT_CONN_CONNECT_ADV);
 		}
+	} else if (!(param->options & BT_LE_ADV_OPT_DIR_MODE_LOW_DUTY)) {
+		start_param.timeout = 128;
```

Comment:
A define might be good for this "magic value"

Reply:
...or at least a code comment explaining it. Could also be both (i.e. a define and a code comment above the define).

Reply:
Put an HCI define in hci.h with a comment.